RecoCTPPS/PixelLocal/src/RPixPlaneCombinatoryTracking.cc

```diff
@@ -3,13 +3,10 @@
 
 #include <algorithm>
 
-
-#include "DataFormats/GeometryVector/interface/GlobalPoint.h"
-
 #include "TMatrixD.h"
 #include "TVectorD.h"
 #include "TVector3.h"
```

Comment:
if I'm not mistaken, the three includes above are not needed

Reply:
You're not. Removed.